### PR TITLE
PHC-718 Added OCR Suggestions Command

### PR DESCRIPTION
**Pull request by cluebbehusen**
This adds an `ocr get-suggestions` command that fetches the unformatted suggestions for the given document and formats them into FHIR.

### CODE REVIEW

File: lib/cmds/ocr_cmds/get-suggestions.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      : undefined;
+  } while (nextPageToken);
+
+  return results;
+};
+
+const resolveValue = (value, resolver) => {
+  const firstIndex = value[0];
+  if (!firstIndex) return null;
+  const extractedValue = firstIndex.value;
```
Comment (rob-snyder) [inline lib/cmds/ocr_cmds/get-suggestions.js:38]: If you want to tighten this up, you could replace:
```
  const firstIndex = value[0];
  if (!firstIndex) return null;
  const extractedValue = firstIndex.value;
```
with
 ```
const extractedValue = value[0] && value[0].value;
```

Just an option, you do you

File: lib/cmds/ocr_cmds/get-suggestions.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    if (extractedCondition.code) {
+      filteredSuggestions.push(extractedCondition);
+    }
+    if (extractedObservation.code) {
+      filteredSuggestions.push(extractedObservation);
+    }
+    if (extractedProcedure.code) {
+      filteredSuggestions.push(extractedProcedure);
+    }
+  });
```
Comment (rob-snyder) [inline lib/cmds/ocr_cmds/get-suggestions.js:160]: So if the user opts for filtered suggestions, then we only want to provide those suggestions that have a code defined for them regardless of whether other fields are defined?

**Review by rob-snyder — APPROVED** [review]
Nice work. Thanks for doing this.

**Review by aniyer — APPROVED** [review]
Great job! thanks..

File: lib/cmds/ocr_cmds/get-suggestions.js (hunk 2 above)
Comment (cluebbehusen) [reply lib/cmds/ocr_cmds/get-suggestions.js:160]: That's what it currently does, but that could be changed. Removing that check results in a lot of resources that just have a date or value field, but without a code field there is no context to what that represents

Comment (rob-snyder) [reply lib/cmds/ocr_cmds/get-suggestions.js:160]: Right. I'm wondering if the medication dosage has more meaning without the coding . . .
I think it's fine for now. We can always change later.

Comment (cluebbehusen) [reply lib/cmds/ocr_cmds/get-suggestions.js:160]: That's a good point. I'll change that

**Review by rob-snyder — APPROVED** [review]

**Review by alexccl — APPROVED** [review]

**Pull request merged**